test/http_client_exception_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Is this a new test? Should the license date be updated?

Reply:
It's copied from dart-lang/sdk